test/workflow_test.go

```diff
@@ -520,6 +520,29 @@ func (w *Workflows) ChildWorkflowCancelUnusualTransitionsRepro(ctx workflow.Cont
 	return nil
 }
 
+func (w *Workflows) ChildWorkflowDuplicatePanicRepro(ctx workflow.Context) error {
+	cwo := workflow.ChildWorkflowOptions{
+		WorkflowID: "ABC-SIMPLE-CHILD-WORKFLOW-ID",
+	}
+	childCtx := workflow.WithChildOptions(ctx, cwo)
+
+	child1 := workflow.ExecuteChildWorkflow(childCtx, w.childWorkflowWaitOnSignal)
+	var childWE workflow.Execution
+	err := child1.GetChildWorkflowExecution().Get(ctx, &childWE)
+	if err != nil {
+		return err
+	}
+	workflow.SignalExternalWorkflow(ctx, childWE.ID, childWE.RunID, "unblock", nil)
+	if err != nil {
+		return err
+	}
+	err = workflow.ExecuteChildWorkflow(childCtx, w.childWorkflowWaitOnSignal).Get(ctx, nil)
+	if _, ok := err.(*internal.ChildWorkflowExecutionAlreadyStartedError); !ok {
```

Comment:
Shouldn't we expose this error outside of internal?

Reply:
Yeah, so, it is one of the many things that's aliased. I can't refer to the `temporal` version of it here b/c circular import. It's aliased here: https://github.com/temporalio/sdk-go/blob/8ab62d9e211fe86c3f5a5723f6c5e79a65f083fd/temporal/error.go#L135

I don't track the whole aliasing stuff so if I need to somehow return it differently I'm all ears.

Reply:
How come other errors can be used from temporal https://github.com/temporalio/sdk-go/blob/master/test/workflow_test.go#L220 but this one can't?

Reply:
Hahah, derp. I didn't realize this comment was on the _test_ file. Fixed.